### PR TITLE
LUCENE-9747: Missing package-info.java causes NPE in MissingDoclet.java

### PR DESCRIPTION
**Pull request by epugh**
# Description

Missing `package-info.java` causes a NPE in the javadocs task and is not obvious what went wrong!

# Solution

Look for the very specific situation, and then don't include the `Element` in the `reporter.print()` method.  Maybe this should be a more generic test, but I wanted to be careful of not breaking other things.

# Tests

Delete a `package-info.java`, and then run javadocs and you'll see the error!

# Checklist

Please review the following and check all that apply:

- [X ] I have reviewed the guidelines for [How to Contribute](https://wiki.apache.org/solr/HowToContribute) and my code conforms to the standards described there to the best of my ability.
- [X ] I have created a Jira issue and added the issue ID to my pull request title.
- [ X] I have given Solr maintainers [access](https://help.github.com/en/articles/allowing-changes-to-a-pull-request-branch-created-from-a-fork) to contribute to my PR branch. (optional but recommended)
- [ X] I have developed this patch against the `master` branch.
- [ X] I have run `./gradlew check`.
- [ ] I have added tests for my changes.
- [ ] I have added documentation for the [Ref Guide](https://github.com/apache/lucene-solr/tree/master/solr/solr-ref-guide) (for Solr changes only).

### CODE REVIEW

**Comment by dweiss** [comment]
Hmmm... something went wrong in this pull request? Why so many commits?

**Comment by epugh** [comment]
I often get this when I branch..   Let me try again.

**Comment by dweiss** [comment]
Strange.  Can you rebase on top of master?

**Comment by epugh** [comment]
@dweiss I ended up making a new PR: https://github.com/apache/lucene-solr/pull/2326

Thanks for looking at this!
